Library/Formula/rswift.rb

```diff
@@ -0,0 +1,17 @@
+class Rswift < Formula
+  desc "Get strong typed, autocompleted resources like images, fonts and segues"
+  homepage "https://github.com/mac-cain13/R.swift"
+  url "https://github.com/mac-cain13/R.swift.git",
+    :tag => "v1.1.1",
+    :revision => "3a6db62164d8f50ccdf43e59894fffb672fd5e3f"
+  depends_on :xcode => "7.0"
+
+  def install
+    xcodebuild "-configuration", "Release", "-scheme", "rswift", "SYMROOT=symroot", "OBJROOT=objroot"
+    bin.install "symroot/Release/rswift"
+  end
+
+  test do
+    system "rswift", "-h"
```

Comment:
this is a outdated diff. please take a look at https://github.com/Homebrew/homebrew/pull/48357/files#diff-4423a6e9c87af8ea0a4fde1153e9e147R16